packages/logic-utils/src/utils/xml-node-to-ui5-node.ts

```diff
@@ -88,6 +92,37 @@ export function getUI5PropertyByXMLAttributeKey(
   return ui5Property;
 }
 
+export function getUI5NodeByXMLAttributeKey(
```

Comment:
we pass the whole attribute here and we really do need the whole attribute (e.g its parent) to implement this.

So perhaps the name should be `getUI5NodeByXMLAttribute`? **without** the `key`